bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -1764,6 +1767,27 @@ public ClientConfiguration setDelayEnsembleChange(boolean enabled) {
         return this;
     }
 
+    /**
+     * Whether to enable bookie address changes tracking.
+     *
+     * @return flag to enable/disable bookie address changes tracking
+     */
+    public boolean getEnableBookieAddressTracking() {
+        return getBoolean(FOLLOW_BOOKIE_ADDRESS_TRACKING, true);
+    }
+
+    /**
+     * Enable/Disable bookie address changes tracking.
+     *
+     * @param enabled
+     *          flag to enable/disable bookie address changes tracking
+     * @return client configuration.
+     */
+    public ClientConfiguration setEnableBookieAddressTracking(boolean enabled) {
```

Comment:
nit: We can use this to enable/disable, so not sure if naming variable as `enabled` is clear. It could just be `value`

Reply:
changed